api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -444,10 +444,13 @@ default void setAttribute(AttributeKey<Long> key, int value) {
     /**
      * Adds a {@link Link} to the newly created {@code Span}.
      *
+     * <p>Links are used to link {@link Span}s in different traces. Used (for example) in batching
```

Comment:
Let's copy this to the other overload too.

Reply:
Done.